buildSrc/src/main/groovy/org/gradle/testretry/build/GradleVersionData.groovy

```diff
@@ -1,10 +1,14 @@
 package org.gradle.testretry.build
 
 import groovy.json.JsonSlurper
-import org.gradle.util.VersionNumber
+import org.gradle.util.GradleVersion
+
+import java.util.regex.Pattern
 
 class GradleVersionData {
 
+    static final Pattern MAJOR_AND_MINOR_VERSION_PATTERN = Pattern.compile("(\\d+)\\.(\\d+).*")
```

Comment:
🧐 Use groovy regex syntax, i.e., `~/regex/ and named groups
```suggestion
    static final Pattern MAJOR_AND_MINOR_VERSION_PATTERN = ~/(?<major>\d+)\.(?<minor>\d+).*/
```

Reply:
DONE (c201b530)